lib/adf_info.py

```diff
@@ -170,25 +170,28 @@ def __init__(self, config_file, debug=False):
 
                 starting_location = Path(baseline_hist_locs)
                 files_list = sorted(starting_location.glob('*'+hist_str+'.*.nc'))
-                base_climo_yrs = sorted(np.unique([i.stem[-7:-3] for i in files_list]))
-
+                base_climo_yrs_str = sorted(np.unique([i.stem[-7:-3] for i in files_list]))
+                base_climo_yrs = []
+                for year in base_climo_yrs_str:
+                   base_climo_yrs.append(int(year))
+                
                 #Check if start or end year is missing.  If so then just assume it is the
                 #start or end of the entire available model data.
                 if syear_baseline is None:
                     print(f"No given start year for {data_name}, using first found year...")
                     syear_baseline = int(base_climo_yrs[0])
-                elif str(syear_baseline) not in base_climo_yrs:
+                elif (syear_baseline) not in base_climo_yrs:
```

Comment:
Probably don't need the parentheses anymore:

```suggestion
                elif syear_baseline not in base_climo_yrs:
```